### PR TITLE
Send SMS on timeout

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
Includes #15 requirements around moving to utils

@smn - ready for review - or I'll ping in prk-dev for cover

File: go-app-chw.js [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-    var EndState = vumigo.states.EndState;
-    var FreeText = vumigo.states.FreeText;
-
-    var GoNDOH = App.extend(function(self) {
-        App.call(self, 'states:start');
-        var $ = self.$;
-
-        self.make_month_choices = function(start, limit) {
+    // make choices options with options
+    make_month_choices: function(Choice, $, start, limit) {
```
Comment (smn) [inline go-app-chw.js]: why the `Choice` parameter here? Why not import it?

File: go-app-chw.js [hunk 2]
```diff
@@ -34,61 +26,119 @@ go.app = function() {
...
+            even += id.charAt(i + 1);
+        }
+        even = '' + even * 2;
+        for (i = 0; even.charAt(i); i++) {
+            c = even.charAt(i);
+            sum += +c;
+        }
+        sum = 10 - ('' + sum).charAt(1);
+        return ('' + sum).slice(-1) == check;
+    },
```
Comment (smn) [inline go-app-chw.js:88]: wow, where did you find the rules for this? For my own peace of mind please link whatever docs you have here.

File: go-app-chw.js [hunk 3]
```diff
@@ -34,61 +26,119 @@ go.app = function() {
...
+
+    is: function(boolean) {
```
Comment (smn) [inline go-app-chw.js]: nit pick on the name, I'd call it `is_true`

File: go-app-chw.js [hunk 4]
```diff
@@ -34,61 +26,119 @@ go.app = function() {
...
-                c = even.charAt(i);
-                sum += +c;
-            }
-            sum = 10 - ('' + sum).charAt(1);
-            return ('' + sum).slice(-1) == check;
+        self.get_finish_reg_sms = function() {
+            return $("Please dial back in to {{ USSD_number }} to complete the pregnancy registration.")
+                .context({
+                    USSD_number: self.im.config.channel
+                });
```
Comment (smn) [inline go-app-chw.js:143]: very nice!

File: go-app-chw.js (hunk 2 above)
Comment (imsickofmaps) [inline go-app-chw.js:88]: http://codereview.stackexchange.com/a/12762

File: go-app-chw.js (hunk 4 above)
Comment (imsickofmaps) [inline go-app-chw.js:143]: The stellar go-di app has provided many hints around what is now possible with v2!

File: go-app-chw.js (hunk 1 above)
Comment (imsickofmaps) [inline go-app-chw.js]: At this point I wasn't groking this utils thing properly and the context - fixed now

**Comment by imsickofmaps** [comment]
Landing this for working on #20 with the share utils

**Pull request merged**